tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -182,4 +180,24 @@ describe(__filename, () => {
     expect(root.find('.AddonMoreInfo-database-id-content'))
       .toHaveText('9001');
   });
+
+  it('links to version history', () => {
+    const addon = createInternalAddon({ ...fakeAddon, slug: 'some-slug' });
+    const root = render({ addon });
+
+    expect(root.find('.AddonMoreInfo-version-history-title'))
+      .toHaveLength(1);
+    const link = root.find('.AddonMoreInfo-version-history-link');
+    expect(link).toHaveProp('href', `/addon/${addon.slug}/versions/`);
+  });
+
+  it('links to development channel versions', () => {
```

Comment:
As mentioned, I think this should check to make sure one exists and render nothing if not.

Also we should call it beta. 😄